packages/salesforcedx-utils-vscode/package.json

```diff
@@ -43,7 +43,7 @@
     "vscode": "^1.26.0"
   },
   "scripts": {
-    "compile": "tsc -p ./",
+    "compile": "webpack",
```

Comment:
Compilation of this module is now done with Webpack and not directly with the Typescript compiler.